### ISSUE TITLE
Sorting declarations in source file

### ISSUE THREAD

**Issue by LisiLisenok** [ceylon/ceylon-ide-common#91]
Since Ceylon allows to have several declarations in a one file it would be nice to sort them somehow. For example: alphabetically, by authors, by `since` annotation, mixing all of those.

**Comment by CPColin** [comment]
I agree. It would be nice if the Organize Imports functionality in Eclipse sorted stuff alphabetically, too, although that may be a separate issue. (It currently sorts the packages alphabetically, but not the stuff being imported from them.)
